src/hotspot/os/windows/park_windows.hpp

```diff
@@ -50,7 +50,8 @@ class PlatformEvent : public CHeapObj<mtSynchronizer> {
     int  fired() { return _Event; }
     void park () ;
     void unpark () ;
-    int  park (jlong millis) ;
+    int  park(jlong millis);
```

Comment:
While you are here could you get rid of the spaces on park/unpark before the parentheses - thanks.

Reply:
Right, did so.